### DISCUSSION TITLE
Best performance for ARM CPU on an embedded device running Ubuntu 22.04?

### DISCUSSION THREAD

**Discussion by kmn1024** [openvinotoolkit/openvino#24540] [Q&A]
I'm wondering if the standard installation via `pip install openvino` will give me the best performance for my setup:
- ARM CPU (A76+A55, which are armv8.2)
- Running on Orange Pi 5, with Ubuntu 22.04
- To do inference on https://github.com/jik876/hifi-gan (a text-to-speech model, which has dynamically-shaped inputs and outputs)

I see many references to "arm" in various cmake files in this repository, and I'm not sure if the standard installation would give me a decent version for my setup, or if there are relatively simple steps I can take to compile my own version.

**Comment by dmitry-gorokhov** [comment]
Hi @kmn1024, starting from openvino 2024.1 release it should be enough to use pip distribution to get best configuration.
If you still interested in building OV from sources: https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md
